fake/src/faker/impls/internet.rs

```diff
@@ -66,7 +66,7 @@ impl<L: Data> Dummy<Username<L>> for String {
             ),
             _ => format!(
                 "{}_{}",
-                FirstName(EN).fake_with_rng::<&str, _>(rng).to_lowercase(),
+                FirstName(c.0).fake_with_rng::<&str, _>(rng).to_lowercase(),
```

Comment:
Why only the default case's FristName use `c.0`?

Reply:
right, nonsense. changed